doc/source/DBEP/DBEP07.rst

```diff
@@ -0,0 +1,60 @@
+===================
+ DBEP 7 : Document mutation
+===================
+
+
+.. contents::
+   :local:
+
+Status
+======
+
+**Discussion**
+
+
+Branches and Pull requests
+==========================
+
+
+Abstract
+========
+The motivation of this DBEP is the handling of document mutation.
+There is a need for post-acquisition modification of metadata.
```

Comment:
Which documents will we allow users to modify?

Reply:
Good point. Only permitted keys in the start and stop documents to begin with.
However, one could imagine mutating the descriptor configuration for ex: the wrong wavelength was given

Reply:
The things in the configuration are usually read from epics.  How do you draw the line between 'correcting' and 'falsifying' data?

Reply:
User defined masks could be generated andgo into the analysisDB. The analysisDB is mostly disposable and has a many to many relationship with the data. If a new mask is needed we can just make a new one.

Wavelengths could also be a generated parameter, since usually some amount of analysis is needed to turn a motor position/xtal scattering into a wavelength.